

message PlayerLoginRequest
{
    required string token = 1;
}

message CreatePlayerRequest{
     optional string nickname = 1;//角色昵称
}

message UpGuideRequest{
	required int32 guide_id = 1;  // 引导ID
}
message NewbeeGuideStepRequest{
	required int32 step_id = 1;
    optional string common_id = 2; // 用于装备，符文，武将等
    optional string sub_common_id = 3; // 用于装备，符文，武将等
}
message ChangeHeadRequest{
	required int32 hero_id = 1;
}
message ChangeBattleSpeed{
	required int32 speed = 1;
}

// 剧情
message ChangeStageStory{
	required int32 story_id = 1;
}

// 显示一次的按钮 command 2203
message ButtonOneTimeRequest{
	required int32 button_id = 1; //0. 第二天活动开启后的按钮  2. 关卡点我有惊喜
}
